stdlib/public/runtime/Reflection.mm

```diff
@@ -1068,62 +1068,57 @@ void swift_ObjCMirror_subscript(String *outString,
   
 // -- MagicMirror implementation.
 
-// TODO: There are other non-Apple platforms that underscore asm symbols.
-#if defined(__APPLE__)
-# define UNDERSCORE "_"
-#else
-# define UNDERSCORE
-#endif
+#define GLUE_EXPANDED(a,b) a##b
+#define GLUE(a,b) GLUE_EXPANDED(a,b)
+#define SYMBOL_NAME(name) GLUE(__USER_LABEL_PREFIX__,name)
```

Comment:
Oh, Im glad to hear that.  Ill make this adjustment shortly.  I think that in the case of MSVC, the macro can be defined by the user (-D__USER_LABEL_PREFIX__=_ or -D__USER_LABEL_PREFIX__= depending on which architecture is being targeted).